src/wallet/db.h

```diff
@@ -100,9 +101,33 @@ class CWalletDBWrapper
     }
 
     /** Create DB handle to real database */
-    CWalletDBWrapper(CDBEnv *env_in, const std::string &strFile_in) :
-        nUpdateCounter(0), nLastSeen(0), nLastFlushed(0), nLastWalletUpdate(0), env(env_in), strFile(strFile_in)
+    CWalletDBWrapper(const fs::path& wallet_path, bool mock = false) :
```

Comment:
> Make the constructors private now?

I didn't do this just because it prevents being able to call MakeUnique so makes code below more verbose. But I think I'll wind up making this tweak if @promag or someone else asks for other style changes on the PR.